src/base/utils/fs.cpp

```diff
@@ -238,40 +239,10 @@ qulonglong Utils::Fs::freeDiskSpaceOnPath(const QString &path)
 {
```

Comment:
I make it return 0 but cause the old check for empty path which returns 0
QStorageInfo::bytesAvailable already returns -1 on failure.
-1 will cause (unknown) when friendly convert it using "friendlyunit" method.
Sorry for not quoting the code, I am currently using mobile.